app/utils/constants.ts

```diff
@@ -59,47 +59,47 @@ export const TIME_OF_DAY_INCREMENTS = [
         dayOfWeek: "Monday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 1.0,
+        multiplier: 1.0,
       },
       {
         name: "Normal Hours",
         dayOfWeek: "Tuesday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 1.0,
+        multiplier: 1.0,
       },
       {
         name: "Normal Hours",
         dayOfWeek: "Wednesday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 1.0,
+        multiplier: 1.0,
       },
       {
         name: "Normal Hours",
         dayOfWeek: "Thursday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 1.0,
+        multiplier: 1.0,
       },
       {
         name: "Normal Hours",
         dayOfWeek: "Friday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 1.0,
+        multiplier: 1.0,
       },
       {
         name: "Weekend Hours",
         dayOfWeek: "Saturday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 2.0,
+        multiplier: 2.0,
       },
       {
         name: "Weekend Hours",
         dayOfWeek: "Sunday",
         startTime: "9:00 AM",
         endTime: "5:00 PM",
-        multipler: 2.0,
+        multiplier: 2.0,
       }] as Ruleset[];
```

Comment:
note: newline here please

Reply:
Unsure what you mean so i put a newline at end of the file, but a lot of our files don't have new lines at the end